src/backend/core/user/views.py

```diff
@@ -115,15 +115,15 @@ class UserViewObs(APIView):
     def get(self, request):
         user = request.user
         if user is None or not user.is_authenticated:
-            return Response({"error": "User does not exist"}, status=400)
+            return Response({"error": "User does not exist"}, status=401)
```

Comment:
Ahh, what a trap 😂

Reply:
Not even the trickiest one. Finding out why context provider is not fetching data in scenario described earlier was the real pain
![cover1](https://github.com/ecoseller/ecoseller/assets/61156764/458854a5-2f82-4386-b1c4-d8a37074be2f)